src/App.js

```diff
@@ -1,12 +1,11 @@
-//import logo from './logo.svg';
 import './App.css';
```

Comment:
if you take advice from above, delete this

Reply:
Whenever I integrate SASS into my app for the color schemes, I'm going to change the file name from "App.css" to "App.scss".  Do you still think I should delete it?